gridstatus/ercot.py

```diff
@@ -3171,7 +3171,11 @@ def parse_doc(
                 # So, it's N during DST And Y during Standard Time
                 # Pandas wants True for DST and False for Standard Time
                 # during ambiguous times
-                ambiguous = doc["DSTFlag"] == "N"
+                # Some ERCOT datasets use a boolean, some use a string
+                if doc["DSTFlag"].dtype == bool:
+                    ambiguous = ~doc["DSTFlag"]
+                else:
+                    ambiguous = doc["DSTFlag"] == "N"
```

Comment:
maybe we can add some defensive programing and assert that unique values are `Y` and `N` here?

i dont feel strongly about this, just trying to think what would have caught this exact error

Reply:
Added a check for 'Y' and 'N' exclusively. Any other changes before merging?